Vagrantfile

```diff
@@ -347,6 +347,13 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         end
         
         config.ssh.insert_key = false
+
+        $enablePswSSH = <<-SCRIPT
+        sudo sed -i "/^[^#]*PasswordAuthentication[[:space:]]no/c\PasswordAuthentication yes" /etc/ssh/sshd_config
```

Comment:
What if `PasswordAuthentication `is not set at all? [The default is no](https://www.freebsd.org/cgi/man.cgi?sshd_config(5)).